<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HisDataDis</class>
 <widget class="QWidget" name="HisDataDis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushCleanButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>420</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushExitButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>420</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>760</width>
     <height>300</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_Title">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>360</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
